CIP-XXXX/README.md

```diff
@@ -0,0 +1,139 @@
+---
+CIP: ????
+Title: Cardano Governance Identifier
```

Comment:
```suggestion
Title: Governance Identifiers
```
For consistency with PR title at this time... removing redundant word `Cardano`, plus characterising a number of representations with plural in title.

Reply:
Maybe even "Governance Credential Identifiers"?

Reply:
@ashisherc I think the latter is better because it's more specific and disambiguates from the many, many types of "identifiers" we will have in the Governance field.

Reply:
This CIP also defines identifier and relevant bech32 prefix for a gov action, which is not a credential, I will keep it just `Governance Identifiers` for now.